common/types/account.go

```diff
@@ -14,7 +14,7 @@ type Account struct {
 	NextNonce       uint64
 	Balance         uint64
 	TemplateAddress *Address
-	State           []byte
+	State           []byte `scale:"max=1024"` // TODO(mafa): check if max. value is correct
```

Comment:
how do you come up with 1024 byte here?

Reply:
Just made up a number, what is a better size?